recipes/CodeLLM_DevKit/validating_code_translation.ipynb

```diff
@@ -0,0 +1,353 @@
+    "        # Get the location of the Java class\n",
+    "        class_path = analysis.get_java_file(qualified_class_name=class_name)\n",
+    "        \n",
+    "        # Read the file content\n",
+    "        if not class_path:\n",
+    "            class_body = \"\"\n",
+    "        with open(class_path, \"r\", encoding=\"utf-8\", errors=\"ignore\") as f:\n",
+    "            class_body = f.read()\n",
+    "        \n",
+    "        # Sanitize the file content by removing comments\n",
+    "        sanitized_class =  JavaSitter().remove_all_comments(source_code=class_body)\n",
+    "\n",
+    "        # Create prompt for translating sanitized Java class to Python\n",
+    "        inst = format_inst(code=sanitized_class, language=\"java\", focal_class=class_name.split(\".\")[-1])\n",
+    "\n",
+    "        print(f\"Instruction:\\n{inst}\\n\")\n",
+    "        print(f\"Translating Java code to Python . . .\\n\")\n",
+    "\n",
+    "        # Prompt the local model on Ollama\n",
+    "        translated_code = prompt_ollama(message=inst)\n",
+    "        \n",
+    "        # Print translated code\n",
+    "        print(f\"Translated Python code: {translated_code}\\n\")\n",
+    "\n",
+    "        # Create python sitter instance for analyzing translated Python code\n",
+    "        py_cldk = PythonSitter()\n",
+    "\n",
+    "        # Compute methods, function, and field counts for translated code\n",
+    "        all_methods = py_cldk.get_all_methods(module=translated_code)\n",
+    "        all_functions = py_cldk.get_all_functions(module=translated_code)\n",
+    "        all_fields = py_cldk.get_all_fields(module=translated_code)\n",
+    "        \n",
+    "        # Check counts against method and field counts for Java code\n",
+    "        if len(all_methods) + len(all_functions) != len(analysis.get_methods_in_class(qualified_class_name=class_name)):\n",
+    "            print(f'Number of translated method not matching in class {class_name}')\n",
+    "        else:\n",
+    "            print(f'Number of translated method in class {class_name} is {len(all_methods)}')\n",
+    "        if all_fields is not None:\n",
+    "            if len(all_fields) != len(analysis.get_class(qualified_class_name=class_name).field_declarations):\n",
+    "                print(f'Number of translated field not matching in class {class_name}')\n",
+    "            else:\n",
+    "                print(f'Number of translated fields in class {class_name} is {len(all_fields)}')\n"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.11.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
```

Comment:
Same issue as above: The last cell just prints `analysis path: .`. 

Could you add a final summary markdown cell that explains what people should have seen?

Reply:
@deanwampler I was able to get output from the last cell.

Reply:
Added final markdown showing expected output